src/generics.jl

```diff
@@ -28,6 +28,13 @@ Base.kron(A::AbstractPDMat, B::AbstractPDMat) = PDMat(kron(Matrix(A), Matrix(B))
 LinearAlgebra.isposdef(::AbstractPDMat) = true
 LinearAlgebra.ishermitian(::AbstractPDMat) = true
 
+"""
+    sqrt(A::AbstractPDMat)
+
+Return the matrix square root of `A`: the matrix `X` such that ``XX = A``
+"""
+LinearAlgebra.sqrt(A::AbstractPDMat) = PDMat(sqrt(Matrix(A)))
```

Comment:
I'm a bit uneasy with this fallback. For special input types usually you don't want a `PDMat` as result, as already PDiagMat and ScalMat show. What's even worse apart from possible inefficiences is that sometimes `PDMat` will even error (e.g., if you use positive-semidefinite matrices as supported and defined in `PDMatsExtras`).

So I think, either we should just use
```suggestion
LinearAlgebra.sqrt(A::AbstractPDMat) = sqrt(Hermitian(Matrix(A)))
```
(by wrapping the matrix with `Hermitian` the eigen decomposition is used: https://docs.julialang.org/en/v1/stdlib/LinearAlgebra/#Base.sqrt-Tuple{StridedMatrix{T}%20where%20T}) or just not define a fallback at all.

Personally, I tend towards the second option.

Reply:
Yep, I'm happy to remove the fallback.